### ISSUE TITLE
Convert immediate numbers to macros

### ISSUE THREAD

**Issue by docularxu** [Linaro/uadk_engine#118]
https://github.com/Linaro/uadk_engine/blob/50692e4bc425c5709a468946cb56ad483be90f7c/src/uadk_digest.c#L819

Should we replace the numbers by the following macros from <openssl/sha.h>?
```
# define SHA224_DIGEST_LENGTH    28
# define SHA256_DIGEST_LENGTH    32
# define SHA384_DIGEST_LENGTH    48
# define SHA512_DIGEST_LENGTH    64
```

**Comment by gaozhangfei** [comment]
yes, it can be changed

**Comment by CaptainMiaow** [comment]
Fixed at c9adda9feccf6f8e6330164ca3900697407d494d
